src/pytorch_lightning/serve/servable_module.py

```diff
@@ -0,0 +1,15 @@
+from modulefinder import Module
```

Comment:
What does this class do and why can't we simply use the built-in protocols for that?